src/main/groovy/org/gradle/android/workarounds/JdkImageWorkaround.groovy

```diff
@@ -54,24 +54,22 @@ class JdkImageWorkaround implements Workaround {
 
         applyToAllAndroidVariants(project) { variant ->
             variant.javaCompileProvider.configure { JavaCompile task ->
-                def jdkImageInput = getJdkImageInput(task)
-                if (jdkImageInput != null) {
-                    setupExtractedJdkImageInputTransform(project, getJvmHome(task))
-                    replaceCommandLineProvider(task, jdkImageInput)
-                }
+                jdkTransform(project, task)
```

Comment:
can we keep the code exactly as before to minimize the diff?

Reply:
done